custom_components/ariston/ariston.py

```diff
@@ -147,6 +147,28 @@ class VelisPlantMode(IntFlag):
     NIGHT = 8
 
 
+@unique
+class ConsumptionType(IntFlag):
+    """Consumption type"""
+
+    HEATING_TOTAL_ENERGY = 1
+    WATER_TOTAL_ENERGY = 2
+    HEATING_GAS = 7
+    WATER_GAS = 10
+    HEATING_ELECTRICITY = 20
+    WATER_ELECTRICITY = 21
+
+
+@unique
+class ConsumptionTimeInterval(IntFlag):
+    """Consumption time interval"""
+
+    TWO_HOURS = 1
+    ONE_DAY = 2
+    TWO_DAYS = 3

```

Comment:
`p:3` seems to be daily value for the past month in the data, not the values for every two days. All the daily values for the last week in `p:2` are repeated in the monthly values from `p:3` in my `consSequencesApi8` data.

Reply:
Damn. You are right. It seems they are different for every devices. So they may don't have exact meaning.

The number of values:
|  | Ariston Genus One | Ariston Alteas One 24 | Velis EVO Wi-fi |
| ------------- | ------------- | ------------- | ------------- | 
| p:1 | 24 | 12 | 24 |
| p:2 | 7 | 7 | 7 |
| p:3 | 57 | 15 | 38 |
| p:4 | 16 | 12 | 13 |